src/nginxconfig/util/backwards_compatibility.js

```diff
@@ -91,6 +91,7 @@ const domainMap = {
     wordpress:              ['php', 'wordPressRules', oldBool],
     drupal:                 ['php', 'drupalRules', oldBool],
     magento:                ['php', 'magentoRules', oldBool],
+    joomla:                 ['php', 'joomlaRules', oldBool],
```

Comment:
```suggestion
```
Joomla was not supported on the old tool, so no backwards compatibility is needed